examples/Mediawiki/Undo.yaml

```diff
@@ -0,0 +1,74 @@
+suite: Undo
+description: Testing undo functionality
+type: RPC
+
+use-fixtures:
+  - Alice
+
+prototypes:
+  edit:
+    - request:
+        method: post
+        path: api.php
+        parameters:
+          action: edit
+          format: json
+      response:
+        body:
+          edit:
+            result: Success
+
+tests:
+  - description: Undo should delete an edit
+    default-session: Alice
+    interaction:
+      # Create page
+      - inherit: edit
+        request:
+          parameters:
+            title: Undo Page
+            createonly: true
+          form-data:
+            summary: create page
+            text: 1
+            token: p/var: Alice/edit-token
+      # Edit page
+      - inherit: edit
+        request:
+          parameters:
+            title: Undo Page
+          form-data:
+            summary: edit page
+            text: 1 2
+            token: p/var: Alice/edit-token
+      # Edit page
+      - inherit: edit
+        request:
+            parameters:
+              title: Undo Page
+            form-data:
+              summary: edit page
+              text: 1 2 4
```

Comment:
Undo only works for entire lines, not parts of lines. It's a lot like git-revert.